CONTRIBUTING.md

```diff
@@ -23,3 +23,13 @@ Before your pull request can be accepted, you must electronically sign the [Ecli
 Unless you are an [elected committer of the GLSP project](https://projects.eclipse.org/projects/ecd.glsp/who), you must include a `Signed-off-by` line in the commit message. This line can be generated with the [-s flag of git commit](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s). By doing this you confirm that your contribution conforms to the Eclipse Contributor Agreement.
 
 For more information, see the [Eclipse Foundation Project Handbook](https://www.eclipse.org/projects/handbook/#resources-commit).
+
+### Branch names and commit messages
+If you are an [elected committer of the GLSP project](https://projects.eclipse.org/projects/ecd.glsp/who) please create a branch. Otherwise please fork and create a branch in your fork for the Pull Request.
+
+The branch name should be in the form `issue/{issue_number}`, e.g. `issue/123` So please create an issue before creating a pull request.
```

Comment:
I'd prefer `issues` instead of `issue`

Reply:
Now my own branch name doesn't match the guidelines ;-)